src/Chapter3.hs

```diff
@@ -480,6 +502,41 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city in total.
 -}
 
+{- ORMOLU_ENABLE -}
+data Castle = MkCastle String
+
+data Wall = BrickWall
+
+data Facility = Church | Library
+
+data People = Citizen
+
+data House = MkHouse [People]
+
+data City = MkCityNoCastle Facility [House] | MkCityWithCastle Castle [Wall] Facility [House]
```

Comment:
The same note about the list of empty `Wall`s. I see that you again need a number 🙂